mungegithub/mungers/fsm/prereview.go

```diff
@@ -28,7 +28,7 @@ const (
 	releaseNote               = "release-note"
 	releaseNoteActionRequired = "release-note-action-required"
 	releaseNoteExperimental   = "release-note-experimental"
-	releaseNoteLabelNeeded    = "release-note-none"
+	releaseNoteNone           = "release-note-none"
```

Comment:
I think not.
We are checking there is there is one of 4 required releaseNote labels.
And releaseNoteLabelNeeded is of other kind. It is to mark that we're missing one of those 4....